templates/index.html

```diff
@@ -5,54 +5,74 @@
         <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" rel="stylesheet">
 {% endblock %}
 {% block body %}
+        <div class="grid-container">
 
-        {% if username %}
-        <p id="greeting">Hello {{ username }}!</p>
-        <div id="login"><a href="{{ url_for('logout') }}">Logout</a></div>
-
-        {% else %}
-        <p id="greeting">You have to log in with Wikidata to edit</p>
-        <div id="login"><a href="{{ url_for('login') }}">Login</a></div>
-        {% endif %}
-        <div id="languagesetting">
-        <label>Language</label>
-        <select id="languageselector">
-                {% for lang in languages %}
-                <option value="{{lang[1]}}" {{'selected' if currentlang == lang[1]}}>{{lang[0]}}</option>
-                {% endfor %}
-        </select>
-        </div>
+                <div id="header">
+                        <p id="info-symbol">&#9432</p>
```

Comment:
Yep, I can do that, and actually a great idea, however could you please provide a screenshot about the whole thing, there could be other inconsistencies. In my own environment it matched everything with the design provided.

Reply:
Even better, you can now look at it yourself: https://tools.wmflabs.org/machtsinn-dev/ ;)

Reply:
I can only check on my mobile now, in firefox looks as on my desktop, except! Login and logout exist at the same time. Yea, to be fair I logged in and saved the whole page, this is how I could create my "environment". Editing html in devtools is pain :D

Reply:
When not logged-in it looks as you described (fine except Login and logout both beein visible), but when logged-in it's pretty broken, probably due to the missing end-if, see below.
To test that, login on the stabel version then switch to dev version.

Reply:
Alright, I'll check it in a few hours, hopefully I'll be able to solve everything

Reply:
Just push to `DDuplinszki:new-design` and ping me, then I'll pull it.

Reply:
@Nudin , ok, I'll push it tomorrow. In theory everything is fixed + the enhancements are done on my prototype.

Reply:
Hi @Nudin , I've pushed to my "new-design" branch, could you please deploy to check if it's working properly?

Reply:
Done. It looks great! Thanks a lot!
The only thing that might still be improved is the placing of the button for medium narrow screens.

I've done a few improvements in the other branch that will hide the Last-saved box until the user actually saved something.

Reply:
I thinks Inkabels idea was to link the lemma and the description to the Lexeme respectively to the Item – that's probably why they are blue in the mokup.

Reply:
Yea, I wanted to finalise the desktop version completely before diving into the responsive side, because I don't really have experience with that.

Could you please elaborate a bit on your 2nd comment about lemma, description etc?

By the way, the dev branch version looks super broken to me.

Reply:
![1](https://user-images.githubusercontent.com/509205/67152364-79aed400-f2d4-11e9-9c10-7158e67b6e1c.png)

Reply:
In what way does dev look broken?

Reply:
If you are ok with it, I would merge this branch, and you create another one for further improvements. This way I can work in parallel on integrating the new features I've been preparing in dev-branch.

Reply:
I see now, the link you provided doesn't contain the new html, I guess because you're working on something else, that's why it looks broken currently. 

Anyhow, sure let's merge this I think, so we now this part is done, then I'll continue slowly with the links and the responsive side of things.